app_utils/sections/experiment.py

```diff
@@ -1734,6 +1734,7 @@ async def experiment_push_to_huggingface_dialog(q: Q, error: str = ""):
                 required=True,
                 tooltip="HF API key, needs write access.",
             ),
+
```

Comment:
@pascal-pfeiffer I should add a ui.checkbox here right? And how do we get a placeholder based on it's value which in turn has to be passed to 
```
        model.backbone.push_to_hub(
            repo_id=repo_id,
            private=True,
            commit_message="Upload model",
            safe_serialization=<placeholder>,
        )
```

PS: Found that to add checkbox in h2o-wave we use ui.checkbox
https://github.com/h2oai/wave/blob/7cf6b29c342771cf0e2a7e81ac93c121715338a1/website/widgets/form/checkbox.md?plain=1#L31

Reply:
yes, you can use `ui.checkbox`, give it a name and use q.client.NAME_OF_CHECKBOX value for your `<placeholder>`. 

`copy_expando(q.args, q.client)` in app.py makes sure that q.client values are always the latest submitted values from q.args